src/grpc.ts

```diff
@@ -100,7 +100,7 @@ function GrpcClient(options) {
     this.grpcVersion = require('grpc/package.json').version;
   }
 }
-module.exports = GrpcClient;
+module.exports = (options) => { return new GrpcClient(options) };
```

Comment:
Please scrutinize this extra carefully :) This is a breaking change. For each of these spots where we export a function / constructor / swampthing, we need to decide if we should export the constructor, or export the instance. 

@stephenplusplus @alexander-fenster I don't know how this library is used, so I probably shouldn't be the one making that call.  What makes the most sense here?